examples/VariantServiceDemo/appsettings.json

```diff
@@ -0,0 +1,45 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
+  "ApplicationInsights": {
+    "ConnectionString": "InstrumentationKey=ccc55c22-7365-4fc4-9033-42c7a6a89327;IngestionEndpoint=https://eastus-8.in.applicationinsights.azure.com/;LiveEndpoint=https://eastus.livediagnostics.monitor.azure.com/;ApplicationId=0491d8e0-b1af-464f-acd4-3b7443d2e630"
```

Comment:
Remove

Reply:
Sorry for the mistake. That app insights store should have been deleted.